index.js

```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```

Comment:
Yeah, I have such thoughts too. Closing this and just adding notes: https://github.com/developit/simple-element-resize-detector/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R40 ?